core/src/test/java/org/apache/druid/timeline/partition/DimensionRangeShardSpecTest.java

```diff
@@ -42,264 +39,6 @@
 
   private final List<String> dimensions = new ArrayList<>();
 
-  @Test
-  public void testIsInChunk()
-  {
-    setDimensions("d1", "d2");
-
-    final DimensionRangeShardSpec shardSpec = new DimensionRangeShardSpec(
-        dimensions,
-        StringTuple.create("India", "Delhi"),
-        StringTuple.create("Spain", "Valencia"),
-        10,
-        null
-    );
-
-    // Verify that entries starting from (India, Delhi) until (Spain, Valencia) are in chunk
-    assertTrue(isInChunk(
-        shardSpec,
-        createRow("India", "Delhi")
-    ));
```

Comment:
These lines (and all the other assertions) could be translated to something like:

```java
assertEquals(
   shard1,
   lookup.getShardSpec(timestamp, createRow("India", "Delhi"))
)
```